src/Tasks/UpdatePackageInfoTask.php

```diff
@@ -101,6 +108,13 @@ public function getDescription()
      */
     public function run($request)
     {
+        // Loading packages and all their updates can be quite memory intensive.
+        $memoryLimit = Config::inst()->get(self::class, 'memory_limit');
+        if (get_increase_memory_limit_max() < translate_memstring($memoryLimit)) {
+            set_increase_memory_limit_max($memoryLimit);
+        }
+        increase_memory_limit_to($memoryLimit);
```

Comment:
If you allow the memory limit prop to be set to false as well, we can allow people to disable this completely. This would be useful when they've decided to set their own memory limit outside of this task in the environment.

It'd probably also be nicer to use default PHP methods to set the memory limit rather than these which look as if they're deprecated framework functions

Reply:
I couldn't tell if they were deprecated so I assumed I should use them. Very happy to not use them as they're a little strange.

Reply:
Oh I see, they're renamed in 4.0.0 (see changelog). In that case feel free to keep using them =)

Reply:
I'm happy to switch to `ini_set` and `ini_get` for actually setting the memory. What do you think I should do?

Reply:
I dunno. If the framework provides supported APIs for this kind of thing then we should probably use them =/

Reply:
Ok I didn't change anything just wrapped it all in an if statement so that setting false will skip any memory setting